Tools/CISupport/ews-build/steps.py

```diff
@@ -667,16 +667,16 @@ def _addToLog(self, logName, message):
 
     def getResultSummary(self):
         if self.results in [FAILURE, SKIPPED]:
-            return {'step': 'Patch doesn\'t have relevant changes'}
+            return {'step': 'Change doesn\'t have relevant changes'}
```

Comment:
"Change doesn't have relevant changes" reads very weird. 
Probably we can have a class variable (string) which stores either "Patch" or "PR" based on the build type, and we print that variable here.